### ISSUE TITLE
Voxel Shape filter

### ISSUE THREAD

**Issue by funabab** [simlu/voxelshop#156]
In the future will you consider supporting this (matching cubes, lego etc) cause i think it pretty cool.

**Comment by simlu** [comment]
The really tricky part is to then also support it for the exporter. I. E. You want the same result when viewing it in blender. But I agree that this would be nice!

**Comment by funabab** [comment]
Yeah export is a bit tricky. But since it's more of like post processing,  i think it will be fair enough to only support dae export since it's the only major generic format that voxelshop (and other 3d program supports).
